debug/bright-v7-types.json

```diff
@@ -0,0 +1,1324 @@
+{
+  "version": 7,
+  "name": "Bright",
+  "constants": {
+    "@name": { "type": "field", "value": "{name_en}" },
```

Comment:
Yeah - this style was slowly slowly manually updated. If you aren't relying on it, I'd like to remove it from this PR and rely on gl-style-migrate for our testing material.